README.md

```diff
@@ -80,165 +81,3 @@ from the command line:
 ```sh
 run app.py
 ```
```

Comment:
Documentation moved to [greaterthan.solutions/pipelayer/framework](https://greaterthan.solutions/pipelayer/framework)